### ISSUE TITLE
Update data accessibility info in Study details & Download tabs, specifying that access requests are pending

### ISSUE THREAD

**Issue by danicahelb** [VEuPathDB/web-eda#1150]
Related to: https://github.com/VEuPathDB/web-eda/issues/1172 and https://github.com/VEuPathDB/EbrcWebsiteCommon/issues/143

Discussed in 5/20/2022 EDA UX meeting. We will add responsive messaging on the download and study details tabs, immediately below the tabs (above all download buttons and links).

We are currently differentiating (1) users who were not logged in / did not have access from (2) users who had access. 

In this ticket, we want to differentiate a third group: (3) users who submitted a request FOR A PROTECTED STUDY but the request is still pending.

For users whose access requests are pending (relevant to protected studies only):
> Your data access request is pending.

_Notes:_ 
_- For public studies, anyone can download the data so we don't need verbiage pending requests._
_- For controlled studies, access requests are approved immediately upon submission and do not need verbiage for pending requests._

**Comment by aurreco-uga** [comment]
in fut (steve, dave and i) on Jan 5 2023, we propose to 
(1) when the user is not approved to access a study, simplify the current verbiage  as:
"To download data please request access. Data will be available upon study team review and approval."
(2) when clicking on the "request access" link, we go to the request form straight (do not show popup)
(3) use the current endpoint permissions to find out if a user has a pending request to access a study:
        ( **api change to be agreed upon**)
       /eda/permissions/
       {  actionAuthorization: {.....}
           **request_pending: true**
           isManager = false
           isUserStudy....
             ....
        }
(4) when the user has already submitted a request and the request is pending, use a new endpoint to grab the request content and show the form filled in (cant be edited): 
      (**endpoint to be agreed upon**)
       /eda/**dataset-request-form**?datasetId=DS_fe95d702c3

**Comment by aurreco-uga** [comment]
In addition to the above Danica notices:
- for prerelease studies we need to get rid of the  sentence (there is a banner already)

**Comment by danicahelb** [comment]
Just to clarify (1): Update the verbiage as follows:

Data Accessibility: Protected
To download data please request access. Data will be available upon study team review and approval.

Data Accessibility: Controlled
To download data please request access. Data will be available upon study team review and approval.

Data Accessibility: Public
Data downloads for this study are public. Data is available without logging in.

Data Accesibility: Prerelease
Data downloads for this study are not yet available on ClinEpiDB.

**Comment by danicahelb** [comment]
Would the current plan mean this ticket is obsolete?

https://github.com/VEuPathDB/EbrcWebsiteCommon/issues/127

**Comment by rwiecknd** [comment]
It appears the backend is ready to support this.

**Comment by aurreco-uga** [comment]
@danicahelb i think the answer is no..  not obsolete

**Comment by danicahelb** [comment]
related to https://github.com/VEuPathDB/EbrcWebsiteCommon/issues/127

**Comment by chowington** [comment]
Blocked by https://github.com/VEuPathDB/service-dataset-access/issues/57

**Comment by chowington** [comment]
> Just to clarify (1): Update the verbiage as follows:
> 
> Data Accessibility: Protected To download data please request access. Data will be available upon study team review and approval.
> 
> Data Accessibility: Controlled To download data please request access. Data will be available upon study team review and approval.
> 
> Data Accessibility: Public Data downloads for this study are public. Data are available without logging in.
> 
> Data Accesibility: Prerelease Data downloads for this study are not yet available on ClinEpiDB.

Hi Danica, do we want the Prelease verbiage to mention ClineEpi specifically? I thought that was something we were avoiding due to code being used in multiple sites

**Comment by danicahelb** [comment]
good catch @chowington! you can say `Data downloads for this study are not yet available on this website.`

**Comment by chowington** [comment]
Not yet done actually---waiting for a review on https://github.com/VEuPathDB/web-eda/pull/1592

**Comment by danicahelb** [comment]
Hey @chowington sorry I messed up in writing this ticket. For controlled studies, the verbiage should be:

Data Accessibility: Controlled
To download data please request access. Data will be available immediately after submitting the request.

**Comment by chowington** [comment]
Fixed by https://github.com/VEuPathDB/web-eda/pull/1676

**Comment by SheenaTomko** [comment]
@danicahelb, this should now be ready for qa again

**Comment by danicahelb** [comment]
looks perfect, thanks @chowington
